packages/dds/tree/src/feature-libraries/editable-tree/utilities.ts

```diff
@@ -0,0 +1,92 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { EmptyKey, Value } from "../../tree";
+import { brand, fail } from "../../util";
+import { TreeSchema, ValueSchema, FieldSchema, LocalFieldKey } from "../../schema-stored";
+// TODO:
+// This module currently is assuming use of defaultFieldKinds.
+// The field kinds should instead come from a view schema registry thats provided somewhere.
+import { fieldKinds } from "../defaultFieldKinds";
+import { FieldKind } from "../modular-schema";
+
+/**
+ * @returns true iff `schema` trees should default to being viewed as just their value when possible.
+ *
+ * Note that this may return true for some types which can not be unwrapped to just their value,
+ * since EditableTree avoids ever unwrapping primitives that are objects
+ * so users checking for primitives by type won't be broken.
+ * Checking for this object case is done elsewhere.
+ */
+export function isPrimitive(schema: TreeSchema): boolean {
+    // TODO: use a separate `TreeViewSchema` type, with metadata that determines if the type is primitive.
+    // Since the above is not done yet, use use a heuristic:
+    return schema.value !== ValueSchema.Nothing &&
+        schema.localFields.size === 0 && schema.globalFields.size === 0;
+}
+
+export type PrimitiveValue = string | boolean | number;
+
+export function isPrimitiveValue(nodeValue: Value): nodeValue is PrimitiveValue {
+    return typeof nodeValue !== "object";
```

Comment:
```suggestion
    return nodeValue !== undefined && typeof nodeValue !== "object";
```

Reply:
Should probably have tests for this function. Adding a utilities.spec.ts for tests for things in this file would be great.

Reply:
done